doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -27,30 +27,29 @@ While this document uses `kubectl` commands for simplicity, working with multipl
 
 ## Prerequisites
 
-This guide expects that you have successfully installed Kuadrant on at least one OpenShift cluster:
-
 - You have completed the steps in [Install Kuadrant on an OpenShift cluster](../install/install-openshift.md) for one or more clusters.
-- For multicluster scenarios, you have installed Kuadrant on at least two different OpenShift clusters, and have a shared accessible Redis store.
-- You have the `kubectl` command line installed.
+- You have the [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) command line tool installed.
+- AWS/Azure or GCP with DNS capabilities.
+- Optional: For multicluster scenarios, you have installed Kuadrant on at least two different OpenShift clusters, and have a shared accessible Redis store.
 - Optional: User workload monitoring is configured to remote write to a central storage system such as Thanos, as described in [Install Kuadrant on an OpenShift cluster](../install/install-openshift.md).
 
-## Platform engineer workflow
+# Platform engineer workflow
 
-NOTE: You must perform the following steps in each cluster individually, unless specifically excluded. 
+**NOTE**: In a multi-cluster scenario, you must perform the following steps in each cluster individually, unless specifically excluded. 
 
 ### Step 1 - Set your environment variables
 
 Set the following environment variables used for convenience in this guide:
 ```bash
-export zid=change-this-to-your-zone-id
-export rootDomain=example.com
-export gatewayNS=api-gateway
-export gatewayName=external
-export devNS=toystore
+export ZONE_ID=change-this-to-your-zone-id
+export ROOT_DOMAIN=example.com
+export GATEWAY_NS=api-gateway
+export GATEWAY_NAME=external
+export DEVELOPER_NS=toystore
 export AWS_ACCESS_KEY_ID=xxxx
```

Comment:
hmm so I am noticing a discrepency between the quick start and this guide. In the quick start we expect `KUADRANT_AWS_ACCESS_KEY_ID` for example but here we remove the Kuadrant. I think it would be best to update the guide to use the ```KUADRANT_AWS_ACCESS_KEY_ID``` type env vars so it flows on more naturally. WDYT

Reply:
at the moment I ahve to alias them to get both to work:

```
export KUADRANT_AWS_DNS_PUBLIC_ZONE_ID=$ZONE_ID
export KUADRANT_AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
export KUADRANT_AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
export KUADRANT_ZONE_ROOT_DOMAIN=cb.hcpapps.net
export KUADRANT_AWS_REGION=$AWS_REGION
```

Reply:
Yeah that sounds good to me